ktor-network/ktor-network-quic/jvmAndNix/src/io/ktor/network/quic/consts/NumericConsts.kt

```diff
@@ -2,10 +2,13 @@
  * Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
  */
 
-package io.ktor.network.quic.util
+package io.ktor.network.quic.consts
```

Comment:
Could you tell me why a separate packet is needed? I would keep it in `quic`

Reply:
I just that I like to keep everything in separate packages and not mix files, easier to navigate for me

Reply:
but maybe for this particular file `util` directory is indeed better